src/main/java/mate/academy/model/Order.java

```diff
@@ -7,7 +7,7 @@ public class Order {
     private List<Product> products;
     private Long userId; // the identifier of user who complete the order
 
-    public Order() {
+    public Order(Long orderId) {
```

Comment:
The `orderId` field is not initialized in this constructor. You should set `this.orderId = orderId;` to properly initialize it.